assets/blocks/index.js

```diff
@@ -7,6 +7,11 @@ import {
 	CourseOutlineLessonBlock,
 	CourseOutlineModuleBlock,
 } from './course-outline';
+import { CourseIcon } from '../icons';
+
+updateCategory( 'sensei-lms', {
+	icon: CourseIcon( { width: '24', height: '24' } ),
```

Comment:
Out of curiosity, do we really need to set the `width` and `height`? I wonder because if this Gutenberg layout changes or something like that, this icon size could change. Or if it's used in other places, too.

Reply:
Yes otherwise the icon takes all available size. Set the size to 20 as it is the size advised in the [handbook](https://developer.wordpress.org/block-editor/developers/filters/block-filters/#managing-block-categories)

Reply:
TIL!